crates/propolis-api-types/src/instance_spec/components/board.rs

```diff
@@ -34,6 +36,58 @@ pub enum Chipset {
     I440Fx(I440Fx),
 }
 
+/// A set of CPUID values to expose to a guest.
+#[derive(Clone, Deserialize, Serialize, Debug, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct Cpuid {
+    /// A list of CPUID leaves/subleaves and their associated values.
+    ///
+    /// The `leaf` and `subleaf` fields of each entry in the template must
+    /// be unique. The leaf must also be in the standard or extended ranges
+    /// (0 to 0xFFFF or 0x8000_0000 to 0x8000_FFFF).
```

Comment:
Having slept on it, this is sort of an awkward spot to refer to other Rust crates, since this appears in the API documentation. I switched this around to refer to the Intel/AMD manuals instead.